src/components/Checkbox/index.stories.tsx

```diff
@@ -34,7 +35,7 @@ export const PreventingUserInteraction: StoryObj<CheckboxProps> = {
       <div>
         <Checkbox
           disabled
-          defaultChecked
+          checked={true}
```

Comment:
not that it matters too much and i also prefer it how you have it right now, but i think the convention in the industry is that you do `<Checkbox disabled checked name="disabled-checked" label="Disabled checked" />`